### PR TITLE
Fix #503: Add endpoint for marking all Inbox messages as read

### LINKED ISSUE #503: Add endpoint for marking all Inbox messages as read
The new endpoint PUT `/inbox/{userId}/messages/read-all` should allow marking all Inbox messages as read for a user.

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

File: powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java [hunk 1]
```diff
@@ -60,6 +62,7 @@ public GetInboxMessageDetailResponse postMessage(String userId, String appId, Cr
         checkAppId(appId);
         final InboxMessageEntity messageEntity = inboxMessageConverter.convert(UUID.randomUUID(), userId, appId, request, new Date());
         final InboxMessageEntity savedMessageEntity = inboxRepository.save(messageEntity);
+        logger.info("Posted new inbox message for user: {}", userId);
```
Comment (banterCZ) [inline powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java:65]: What about to log even message id?

**Review by banterCZ — APPROVED** [review]
Looks good.

File: powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java [hunk 2]
```diff
@@ -102,13 +105,20 @@ public GetInboxMessageDetailResponse readMessage(String userId, String appId, St
         final InboxMessageEntity inboxMessage = messageEntity.get();
         if (!inboxMessage.isRead()) { // do not call repository save if there is no change.
             inboxMessage.setRead(true);
+            logger.info("Marked imbox message as read for user: {}, ID: {}", userId, inboxId);
```
[the remark this answers is not in the document]
Comment (banterCZ) [reply powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java:108]: What about to log even message id?

File: powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java (hunk 1 above)
Comment (romanstrobl) [reply powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java:65]: Sure.

File: powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (romanstrobl) [reply powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java:108]: Sure.

**Pull request merged**